sc_preprocessing/2_make_pseudobulks/scanpy_sc_preprocess_allSplits-liver.ipynb

```diff
@@ -0,0 +1,1899 @@
+{
```

Comment:
<p>this is very nice! as a comment, when i have used your code in this format, i typically separate the "long" pseudobulks-making loop from everything else. like, i like to QC and see my cell_order, len_vector beforehand. so i print it in a kernel before the loop so i dont make a bunch of files with the wrong cells. just an FYI.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/buddi_analysis/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>ah, interesting. I will think about making it more modular, thanks for letting me know!</p>